ci/do_ci.sh

```diff
@@ -40,6 +40,33 @@ elif [[ "$1" == "cmake.exporter.otprotocol.test" ]]; then
   make
   make test
   exit 0
+elif [[ "$1" == "cmake.exporter.prometheus.test" ]]; then
+#  export DEBIAN_FRONTEND=noninteractive
+#  apt-get update
+#  apt-get install sudo
+#  apt-get install zlib1g-dev
+#  apt-get -y install libcurl4-openssl-dev
+  cd third_party
+  git clone https://github.com/jupp0r/prometheus-cpp
```

Comment:
Consider extract this logic to a separate file under the Prometheus exporter folder.

Reply:
Sure, correct me if I misunderstand something:
1. keep the `elif [[ "$1" == "cmake.exporter.prometheus.test" ]];` block here
2. move the scripts of installing the Prometheus client (from `cd third_party` to `sudo make install`) to a separate script in `exporters/prometheus`. For example, `exporters/prometheus/install_prometheus_client.sh`
3. run `install_prometheus_client.sh` here